### ISSUE TITLE
Typo

### ISSUE THREAD

**Issue by AjinkyaDhote** [MicrosoftDocs/cpp-docs#739]
However, some non-European character sets, such as Japanese Kanji, include many more characters than can &lt;- ( should be cannot ) 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d4d356b2-081d-1728-6eb9-e0fcce6caa82
* Version Independent ID: 68163ebe-27d4-5c4f-2503-d40584df2556
* Content: [Single-Byte and Multibyte Character Sets](https://docs.microsoft.com/en-us/cpp/c-runtime-library/single-byte-and-multibyte-character-sets?view=vs-2017)
* Content Source: [docs/c-runtime-library/single-byte-and-multibyte-character-sets.md](https://github.com/Microsoft/cpp-docs/blob/master/docs/c-runtime-library/single-byte-and-multibyte-character-sets.md)
* Product: **visual-cpp**
* GitHub Login: @corob-msft
* Microsoft Alias: **corob**

**Comment by AjinkyaDhote** [comment]
Yeah Exactly, But the sentence "However, some non-European character sets, such as Japanese Kanji, include many more characters than 'can' be represented in a single-byte coding scheme, and therefore require multibyte-character set (MBCS) encoding.  
Notice the string 'can be represented'. If it can be there is no need for MBCS. It 'cannot be represented in SBCS' that is the reason it needs MBCS. 
'can' should be replaced by 'cannot' also 'than should be that'

**Comment by corob-msft** [comment]
"Than can" works with the comparator "more" in a way that "that can't" doesn't. "John has more dogs than can be put in his car" is similar to but not the same as "John has more dogs that can't be put in his car".

**Comment by AjinkyaDhote** [comment]
Okay, Thanks for the patience and the explanation 👍
